index.html

```diff
@@ -10,8 +10,8 @@
 
     <nav id="navbar">
       <ul class="nav-list">
-       <li> <a href="#welcome">Home</a></li>
-       <li> <a href="#projects">Work</a></li>
+       <li> <a href="#welcome" alt="home">Home</a></li>
```

Comment:
Kindly add Alt to the image tag to provide alternative information for the visually impaired users to know what is the position of the image as well as for screen readers. alt tag gives a brief (and often different) textual representation of what your website is all about.